src/app/routes/getInitialData/index.js

```diff
@@ -48,9 +56,9 @@ const handleError = error => {
   return { error, status: STATUS_CODE_BAD_GATEWAY };
 };
 
-const fetchData = ({ pathname, preprocessorRules }) =>
+const fetchData = pathname =>
   fetch(getUrl(pathname)) // Remove .amp at the end of pathnames for AMP pages.
-    .then(handleResponse(preprocessorRules))
+    .then(handleResponse())
```

Comment:
handleResponse doesn't need to be curried anymore

Reply:
Hmmm trying this local and it all fails, not sure why

Reply:
Missed the removal of `() =` in `handleResponse` 🤦‍♂  working now